JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+msgid "95% CI Lower and Upper Bound"
+msgstr "95% Geloofwaardigheidsinterval Onder- en Bovengrens"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "N"
+msgstr "N"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "N (group 1)"
+msgstr "N (groep 1)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "N (group 2)"
+msgstr "N (groep 2)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Study Labels"
+msgstr "Onderzoeksnamen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "One-sample t-tests"
+msgstr "T-toets voor één gemiddelde"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Two-sample t-tests"
+msgstr "Onafhankelijke steekproeven t-toets"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Priors"
+msgstr "Priors"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Plot priors"
+msgstr "Priorsgrafiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Effect"
+msgstr "Effect"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Distribution"
+msgstr "Verdeling"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Parameters"
+msgstr "Parameters"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Truncation"
+msgstr "Afkapping"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Prior Odds"
+msgstr "Prior kansratio"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Normal(μ,σ)"
+msgstr "Normaal(μ,σ)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Student-t(μ,σ,v)"
+msgstr "Student-t(μ,σ,v)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Cauchy(x₀,θ)"
+msgstr "Cauchy(x₀,θ)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Gamma(α,β)"
+msgstr "Gamma(α,β)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Gamma(k,θ)"
+msgstr "Gamma(k,θ)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Inverse-Gamma(α,β)"
+msgstr "Inverse-Gamma(α,β)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Spike(x₀)"
+msgstr "Puntmassa((x₀)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Uniform(a,b)"
+msgstr "Uniform(a,b)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "lower"
+msgstr "onder"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "upper"
+msgstr "boven"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Odds"
+msgstr "Kansratio"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Heterogeneity"
+msgstr "Heterogeniteit"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Publication bias"
+msgstr "Publicatieonzuiverheid"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Weight function"
+msgstr "Gewichtsfunctie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Cut-points"
+msgstr "Afkap punt"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Two-sided"
+msgstr "Twee-zijdig"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "One-sided (mon.)"
+msgstr "Een-zijdig (mon.)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "One-sided"
+msgstr "Een-zijdig"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "None"
+msgstr "Geen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "p-values"
+msgstr "p-waarden"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Set null priors"
+msgstr "Zet null priors"
```

Comment:
Hebben we niets voor `null`?

Reply:
Volgens mij kunnen we gewoon null model zeggen, maar ik weet het niet zeker. Na een tijdje ben ik gestopt met Prior and Posterior vertalen naar A Priori en A Posteriori